src/screens/levelOverview.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {SectionList, StyleSheet, Text, View} from 'react-native';
+import {DATA_LEVEL_OVERVIEW} from '../consts/lessons';
```

Comment:
let's not have a space between all the react-* imports, then 1 space before utils/helpers/consts/...